Check if domain is set for terms of service

xh3n1 opened the pull request:
As discussed here https://github.com/phpList/phplist3/pull/301
Also made an improvement regarding getNewAttributeTablename.

samtuke commented:
Please can you remind us of the purpose of this PR (which discussion within #431 is referred to)?

xh3n1 commented:
@samtuke Since we use the domain placeholder for the terms of service, I added a check if it is set or not, and if it is not set I replaced the "domain" placeholder with "our website" text.

samtuke commented:
@xh3n1 Thanks!
